app/views/settings/index.html.erb

```diff
@@ -108,10 +108,6 @@
 			<section class='settings-pane nonprofit-settings social-links hide'>
 				<%= render 'social_links' %>
 			</section>
-
-			# <section class='settings-pane nonprofit-settings tracking hide'>
-			# 	<%= render 'custom_tracking' %>
-			# </section>
```

Comment:
We want the tab fully removed, not commented out. The goal is to have nothing in the html, views and javascript related to this custom_tracking functionality.